app/base/file_handler.py

```diff
@@ -17,40 +21,57 @@ def create_images_folder(username):
     return property_img_folder
 
 
-def property_image_handler(username, image_files, save_to_folder):
+def save_images_to_temp_folder(image_files):
     """
-    Handles the docs uploaded from the web form. The docs are down sized using
-    the Pillow image library and saved to the file system. The image filenames are checked
-    using the werkzeug utilities, then the filename is saved to the list of filenames in the
-    dictionary object.
+    Generates a temporary folder for storing image files of property.
     """
-    images_list = [save_to_folder]
+    rand_dir_name = secrets.token_hex(16)
+    temp_dir = rand_dir_name
+    os.mkdir(f"{current_app.root_path}/base/static/property_images/{temp_dir}")
+
     for image_file in image_files:
+        filename_hex = secrets.token_hex(10)
         checked_filename = secure_filename(image_file.filename)
-        random_hex = secrets.token_hex(15)
-        _, file_extension = os.path.splitext(checked_filename)
-        clean_filename = f"{username}{random_hex}{file_extension}"
-        images_list.append(clean_filename)
+        _, file_ext = os.path.splitext(checked_filename)
+        checked_filename = f"{filename_hex}{file_ext}"
```

Comment:
same here I would encourage that we migrate gradually from using hex

there is a rather convinient library for random elements 
called uuid

e.g

```
>>> import uuid

>>> suffix = str(uuid.uuid4())
>>> # this  will give you back a 36 bite uniq object that you can end  up slicing to your  liking
>>> ' 9ea22191-b38a-4ac3-b623-7d1092e887a4'
>>> # for example if you just want to use a few chars

>>> suffix[:8] 
>>> 44eccf40
# OR
>>> suffix.split('-')[0]
>>> 44eccf40

Reply:
Okay noted